packages/database/src/exp/DataSnapshot.ts

```diff
@@ -15,54 +15,90 @@
  * limitations under the License.
  */
 
-import { Reference } from './Reference';
+import { child, Reference } from './Reference';
+import { Node } from '../core/snap/Node';
+import { Index } from '../core/snap/indexes';
```

Comment:
I think this is breaking the build because I think it wants `indexes/Index` and I guess this shortcut works if a file is named `index` but I think it might be case sensitive. Didn't go through the rest of the PR yet, just wanted to put in this comment first because I think it's what's breaking the build.

Reply:
Thanks. Fixed this and the tests are passing. I can't come up with a better name for this file, otherwise I would have changed it long ago.